From 9c01fc77723c5d5c8c47ee799ed79ad5611e868f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Steven!=20Ragnar=C3=B6k?= <steven@nuclearsandwich.com>
Date: Thu, 15 Jun 2017 16:13:32 -0400
Subject: [PATCH 3/4] Fix args format for dh_shlibdeps.

We were getting lucky here. dpkg-shlibdeps takes multiple -l args but
dh_shlibdeps only takes one with paths separated by colons. The last
one, which was the needed one, was winning but if the package ever had
shared libraries that it built and relied on itself this would bust.
---
 debian/rules.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 8ef84e5..6c0eae7 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -53,7 +53,7 @@ override_dh_shlibdeps:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
-	dh_shlibdeps -l$(CURDIR)/debian/@(Package)/@(InstallationPrefix)/lib/ -l@(InstallationPrefix)/lib/
+	dh_shlibdeps -l$(CURDIR)/debian/@(Package)/@(InstallationPrefix)/lib/:@(InstallationPrefix)/lib/
 
 override_dh_auto_install:
 	# In case we're installing to a non-standard location, look for a setup.sh
-- 
2.7.4

